api/controllers/calendarEntry.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response, NextFunction } from 'express';
+
+interface CalendarEntry {
+    id: string
+    eventId: string
+    creatorId: string
+    title: string
+    description: string
+    isAllDay: boolean
+    startTimeUtc: Date
+    endTimeUtc: Date
+    createdAt: Date
+    updatedAt: Date
+  };
+    
+  export const getCalendarEntries = (request: Request, response: Response, next: NextFunction) => {
+    const date = new Date()
+    let entries: CalendarEntry[] = [
+      {
+        id: "id",
+        eventId: "eventId",
+        creatorId: "creatorId",
+        title: "title",
+        description: "description",
+        isAllDay: false,
+        startTimeUtc: date,
+        endTimeUtc: date,
+        createdAt: date,
+        updatedAt: date,
+      },
+    ];
```

Comment:
Could you explain why this is a `let` rather than a `const` variable if we're assigning a value to it right away?

Reply:
It can be a const - will update.